### ISSUE TITLE
At sign '@' is passed thru in constructor

### ISSUE THREAD

**Issue by darkoverlordofdata** [eventualbuddha/decaffeinate#47]
this 

```coffee
class Scooby
  constructor: (@doo=42) ->
```

is converted to this invalid es6 syntax

```js
class Scooby {
  constructor(@doo=42) {}		
}
```

**Comment by eventualbuddha** [comment]
Thanks! This is partially addressed by v1.26.1 and will now be transformed into this:

```js
class Scooby {
  constructor(this.doo=42) {}
}
```

This is still not valid ES6 syntax. However, the fix means that things like this will now work:

```coffee
(a=b[0..2]) ->
```

**Comment by eventualbuddha** [comment]
Fixed completely in v1.27.2:

```coffee
class Scooby
  constructor: (@doo=42) ->
```

```js
class Scooby {
  constructor(doo=42) { this.doo = doo; }
}
```
